spring-cloud-aws-sqs/src/test/java/io/awspring/cloud/sqs/integration/SqsIntegrationTests.java

```diff
@@ -231,6 +251,52 @@ void doesNotAckOnErrorBatchAsync() throws Exception {
 		assertThat(latchContainer.doesNotAckBatchAsyncLatch.await(10, TimeUnit.SECONDS)).isTrue();
 	}
 
+	@Test
```

Comment:
I think this test suite is already too big for us to introduce this many tests.

Would you mind moving these tests to their own suite?